src/win/core.c

```diff
@@ -80,6 +81,38 @@ static void uv__crt_invalid_parameter_handler(const wchar_t* expression,
 }
 #endif
 
+static QUEUE uv_loops;
+static uv_mutex_t uv_loops_lock;
+
+static void uv__loops_init() {
+  uv_mutex_init(&uv_loops_lock);
+  QUEUE_INIT(&uv_loops);
+}
+
+static void uv__loops_add(uv_loop_t* loop) {
+  uv_mutex_lock(&uv_loops_lock);
+  QUEUE_INIT(&loop->uv_loops);
```

Comment:
Isn't this the way it should be done? Initialize it before adding it to uv_loops (line 84)?